core/src/main/java/org/apache/iceberg/PartitionsTable.java

```diff
@@ -36,6 +36,8 @@ public class PartitionsTable extends BaseMetadataTable {
 
   private final Schema schema;
 
+  private final boolean isUnpartitionedTable;
```

Comment:
Nit: can we remove is?  Maybe just 'unpartitioned'?   From: https://iceberg.apache.org/contribute/#java-style-guidelines

Reply:
updated.